deps/Chakra.ICU/Chakra.ICU.Build.props

```diff
@@ -20,6 +20,12 @@
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
 
+      <!-- Disable renaming to maintain compatibility with Windows Kit ICU's icuuc/icuin.lib -->
+      <PreprocessorDefinitions>
+        U_DISABLE_RENAMING=1;
```

Comment:
>U_DISABLE_RENAMING=1; [](start = 8, length = 21)

Redundant with commondefines.h?

Reply:
This is intentionally specific to Chakra.ICU while CommonDefines was intentionally specific to ChakraCore, though I suppose this specific flag could be combined.